post/migrations/0004_auto_20190509_2008.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2 on 2019-05-09 20:08
```

Comment:
[minor]
Migrations should have more descriptive names. I believe you can generate them with a name with a `-n` argument when calling `makemigrations`.